clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10406,6 +10406,42 @@ void OffloadDeps::ConstructJobMultipleOutputs(Compilation &C,
   constructJob(C, JA, Outputs, Inputs, TCArgs, LinkingOutput);
 }
 
+// Utility function to gather all options for LLVM to SPIR-V translation using
+// SPIR-V backend.
+static void getSPIRVBackendOpts(const llvm::opt::ArgList &TCArgs,
+                                ArgStringList &TranslatorArgs) {
+  TranslatorArgs.push_back(TCArgs.MakeArgString("-filetype=obj"));
+  TranslatorArgs.push_back(
+      TCArgs.MakeArgString("-mtriple=spirv64-unknown-unknown"));
+  TranslatorArgs.push_back(TCArgs.MakeArgString("-O0"));
+  TranslatorArgs.push_back(
+      TCArgs.MakeArgString("--avoid-spirv-capabilities=Shader"));
+  TranslatorArgs.push_back(
+      TCArgs.MakeArgString("--translator-compatibility-mode"));
+
+  // Disable all the extensions by default
+  std::string ExtArg("-spirv-ext=");
```

Comment:
I think there is a typo? Missing one `-` in `--spirv-ext`. Please see [an example SPIR-V backend LIT test](https://github.com/llvm/llvm-project/blob/main/llvm/test/CodeGen/SPIRV/extensions/SPV_INTEL_arbitrary_precision_integers.ll).